webpack.config.js

```diff
@@ -244,6 +244,8 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 					'react-virtualized': 'react-virtualized/dist/commonjs',
 					'social-logos/example': 'social-logos/build/example',
 					debug: path.resolve( __dirname, 'node_modules/debug' ),
+					'@calypstyle': path.resolve( path.join( __dirname, 'assets', 'stylesheets', 'shared' ) ),
```

Comment:
Don't forget to pick an appropriate name for this! 😄 
`@calypso-style` sounds better, but we should probably be more explicit in the fact that it targets the `shared` folder, and not any scss file.
What about `@calypso-style-shared` or `@calypso-style-common`? Are they too much of a mouthful?